specification/confidentialledger/Microsoft.CodeTransparency/models.tsp

```diff
@@ -0,0 +1,275 @@
+  kty: string;
+
+  @doc("RSA modulus, in Base64")
+  n?: string;
+
+  @doc("RSA secret prime")
+  p?: string;
+
+  @doc("RSA secret prime, with p < q")
+  q?: string;
+
+  @doc("RSA Private Key Parameter")
+  qi?: string;
+
+  @doc("Use (\"public key use\") identifies the intended use of\nthe public key. The \"use\" parameter is employed to indicate whether\na public key is used for encrypting data or verifying the signature\non data. Values are commonly \"sig\" (signature) or \"enc\" (encryption).")
+  use?: string;
+
+  @doc("X coordinate for the Elliptic Curve point")
+  x?: string;
+
+  @doc("The \"x5c\" (X.509 certificate chain) parameter contains a chain of one\nor more PKIX certificates [RFC5280].  The certificate chain is\nrepresented as a JSON array of certificate value strings.  Each\nstring in the array is a base64-encoded (Section 4 of [RFC4648] --\nnot base64url-encoded) DER [ITU.X690.1994] PKIX certificate value.\nThe PKIX certificate containing the key value MUST be the first\ncertificate.")
+  x5c?: Array<string>;
+
+  @doc("Y coordinate for the Elliptic Curve point")
+  y?: string;
+}
+
+#suppress "@azure-tools/typespec-azure-core/casing-style"
+@doc("The Response body of the Version API.")
+model VersionResult {
+  @doc("Version of the CodeTransparency service")
+  scitt_version: string;
+}
+
+#suppress "@azure-tools/typespec-azure-core/documentation-required"
+#suppress "@azure-tools/typespec-azure-core/casing-style"
+@doc("The Response body of the Configuration API.")
+model CodeTransparencyConfiguration {
+  policy?: CodeTransparencyConfigurationPolicy;
+  authentication?: CodeTransparencyConfigurationAuthentication;
+
+  @doc("did:web identifier")
+  @pattern("^did:web:.+$")
+  service_identifier?: string;
+}
+
+#suppress "@azure-tools/typespec-azure-core/documentation-required"
+#suppress "@azure-tools/typespec-azure-core/casing-style"
+@doc("Configuration of CCF authentication.")
+model CodeTransparencyConfigurationAuthentication {
+  allow_unauthenticated: boolean;
+  jwt?: CodeTransparencyConfigurationAuthenticationJwt;
+}
+
+#suppress "@azure-tools/typespec-azure-core/documentation-required"
+#suppress "@azure-tools/typespec-azure-core/casing-style"
+@doc("Configuration of JWT claims")
+model CodeTransparencyConfigurationAuthenticationJwt {
+  required_claims: Record<string>;
+}
+
+#suppress "@azure-tools/typespec-azure-core/documentation-required"
+#suppress "@azure-tools/typespec-azure-core/casing-style"
+@doc("Configuration of CCF Authentication policies.")
+model CodeTransparencyConfigurationPolicy {
+  accepted_algorithms?: string[];
+  accepted_did_issuers?: string[];
+}
```

Comment:
Please make sure you would fix the casing in next api-version (or provide a convincing reason that it had to stay snake_case).

Reply:
I've raised it as a separate issue and we'll track and fix it in a later version https://github.com/microsoft/scitt-ccf-ledger/issues/187